Project.toml

```diff
@@ -32,7 +32,7 @@ ForwardDiff = "0.10"
 GPLikelihoods = "0.4"
 IrrationalConstants = "0.1"
 LogExpFunctions = "0.3"
-PDMats = "0.11"
+PDMats = "< 0.11.8"
```

Comment:
This is probably not what you want, for two reasons:
- It adds support for PDMats 0.1, 0.2, ..., 0.10
- The update is not guaranteed to reach users since it involves downgrading PDMats for users and packages that already use PDMats 0.11.8 and hence might not be performed by Pkg

Reply:
We should then use a hyphen "0.11.0 - 0.11.7"?

My understanding is if someone uses multiple packages that depend on PDMats, some of which are compatible with 0.11.8, and then they add ApproximateGPs.jl, Pkg will resolve to 0.11.7. But maybe I misunderstood what you mean.

Please feel free to propose any alternative quick fixes. The fact is that with 0.11.8 this package fails quite badly. I think fixing it properly will take a bit longer (and I'm unable to do this right now). Limiting to PDMats v0.11.7 restores the intended behavior, I think.